pinot-common/src/main/java/org/apache/pinot/common/metrics/AbstractMetrics.java

```diff
@@ -732,14 +729,4 @@ private void removeGaugeFromMetricRegistry(String metricName) {
   protected String getTableName(String tableName) {
     return _isTableLevelMetricsEnabled || _allowedTables.contains(tableName) ? tableName : "allTables";
   }
-
-  /**
```

Comment:
Should removeGauge be deprecated?

Reply:
Nope, it's still useful for removing gauge. And we keep `_gaugeValues.remove(gaugeName);` for backward compatibility.

Reply:
ok